### ISSUE TITLE
 Ajouter une procédure préalable pour avoir LaTeX sur le poste.

### ISSUE THREAD

**Issue by jengelaere** [MTES-MCT/parcours_r_module_publication_rmarkdown#13]
Sinon les output pdf sortent pas

**Comment by jdlom** [comment]
Oui, il faut faire référence à ceci : https://bookdown.org/yihui/rmarkdown-cookbook/install-latex.html

**Comment by jdlom** [comment]
Oui, il faut faire référence à ceci : https://bookdown.org/yihui/rmarkdown-cookbook/install-latex.html

**Comment by MaelTheuliere** [comment]
Pourquoi y a t il besoin de latex pour chrome_print ?

**Comment by jengelaere** [comment]
Coucou Maël, Franck est en train de tester les supports de formation, il semble qu'il y traine des rmarkdown dont le yaml contient "output:   pdf_document: default. 
l'autre option serait peut être de tout convertir à pagedown
